ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml

```diff
@@ -1,9 +1,27 @@
+base_images:
+  golangci-lint:
+    name: golangci-lint
+    namespace: ci
+    tag: v1.46.0
+  ocp_builder_rhel-8-golang-1.18-openshift-4.12:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.18-openshift-4.12
 binary_build_commands: make manager
 build_root:
   image_stream_tag:
     name: release
     namespace: openshift
     tag: golang-1.18
+images:
+- dockerfile_path: Dockerfile.must-gather
+  to: must-gather
```

Comment:
What about `kernel-module-management` image? I don't think the mirroring will work if we don't build the image and push it to the internal CI image registry.

Reply:
It should be built automatically already.

Reply:
How? In d/s we mention it explicitly https://github.com/openshift/release/blob/f00cd51e53109cbfadee8246f00fa403806c0ba3/ci-operator/config/openshift/special-resource-operator/openshift-special-resource-operator-master.yaml#L16-L22.

I don't see how the image will build without it?

Eventually, there is a job called `images` that runs on PRs, this jobs build the images from that section and other test will fail (and probably aborted) if that job failed.

So in addition to the mirroring, we also want that validation in CI.

Reply:
It *should* build `registry.ci.openshift.org/edge-infrastructure/kernel-module-management`. I do not think we need to do what we are doing with SRO.
I suggest we merge to confirm.

Reply:
> I do not think we need to do what we are doing with SRO.

What are we doing in SRO that we want to skip here?

ATM, we are going to have that `images` job anyway - it is added by the CI operator. It will test the must gather image but won't test the KMM image build as well. What it means is that we have a partial infrastructure (missing 50% of the images we are promoting).